.pkg/io.github.jpochyla.Psst.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.jpochyla.Psst</id>
+  <url type="homepage">https://github.com/jpochyla/psst</url>
+
+  <name>Psst</name>
+  <summary>Fast Spotify client with native GUI, without Electron, built in Rust.</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <description>
+    <p>
+      Fast Spotify client with native GUI, without Electron, built in Rust. Very early in development, lacking in features, stability, and general user experience. It is being tested only on Mac so far, but aims for full Windows and Linux support. Contributions welcome!
```

Comment:
The "It is being tested only on Mac so far, but aims for full Windows and Linux support." sentence should be removed since Flatpak is only on Linux anyway.

Reply:
This file is unlerated to flatpak, it is the app's metainfo.

Reply:
That's ok. We've since updated that sentence in the readme so it should be updated here anyways.